modules/afuser/afuser.c

```diff
@@ -189,5 +192,7 @@ afuser_dd_new(gchar *user, GlobalConfig *cfg)
   self->super.super.super.queue = afuser_dd_queue;
   self->super.super.super.free_fn = afuser_dd_free;
   self->username = g_string_new(user);
+  self->time_reopen = cfg ? cfg->time_reopen : 60;
```

Comment:
Small typo here. Otherwise LGTM!

Reply:
Thanks for the review, I addressed the problems with the default value in another comment: https://github.com/syslog-ng/syslog-ng/pull/3585#discussion_r582613561